data/brands/shop/beverages.json

```
@@ -188,6 +188,16 @@
         "shop": "beverages"
       }
     },
+    {
+      "displayName": "Machi machi",
+      "tags": {
+        "brand": "麥吉 machi machi",
```

```suggestion
        "brand": "machi machi",
        "brand:en": "machi machi",
        "brand:zh": "麥吉",
```
I looked their website, only 2 shop in Taiwan, most of their branch shop are all over the world, so I think this is a 001 entry, so use English as brand only.

Besides, according to photo on its website, they only use "machi machi"